routes/web.php

```diff
@@ -13,25 +13,30 @@
 |
 */
 
-Route::get('/articles',             'ArticleController@index')->name('index');
-Route::get('/stock',                'ArticleController@stock')->name('stock');
-Route::get('/articles/create',      'ArticleController@create')->name('create');
-Route::post('/articles',            'ArticleController@store');
-Route::get('/articles/{article}',      'ArticleController@show')->name('show');
+Route::get('/articles',                              'ArticleController@index')->name('index');
+Route::get('/stock',                                 'ArticleController@stock')->name('stock');
+Route::get('/articles/create',                       'ArticleController@create')->name('create');
+Route::post('/articles',                             'ArticleController@store');
+Route::get('/articles/{article}',                    'ArticleController@show')->name('show');
 
-Route::get('/articles/{article}/edit',      'ArticleController@edit');
-Route::patch('/articles/{article}',      'ArticleController@update');
-Route::delete('/articles/{article}',      'ArticleController@destroy');
+Route::get('/articles/{article}/entries',            'ArticleController@entries')->name('entries');
+Route::get('/articles/{article}/consumptions',       'ArticleController@consumptions')->name('consumptions');
 
-Route::post('/consumptions',        'ConsumptionController@store');
+Route::get('/articles/{article}/edit',               'ArticleController@edit');
+Route::patch('/articles/{article}',                  'ArticleController@update');
+Route::delete('/articles/{article}',                 'ArticleController@destroy');
 
-Route::post('/entries',             'EntryController@store');
-Route::get('/entries/index',             'EntryController@index');
-Route::get('/entries/{entry}',      'EntryController@show');
 
-Route::get('/entries/{entry}/edit',      'EntryController@edit');
-Route::patch('/entries/{entry}',      'EntryController@update');
-Route::delete('/entries/{entry}',      'EntryController@destroy');
+Route::post('/entries',                            'EntryController@store');
+Route::get('/entries/{entry}/edit',                'EntryController@edit');
+Route::patch('/entries/{entry}',                   'EntryController@update');
+Route::delete('/entries/{entry}',                  'EntryController@destroy');
+
+Route::post('/consumptions',                       'ConsumptionController@store');
```

Comment:
In other files i have left some comments about how to write your routes. With these comments i thought you were using reource-controllers. I now see that this is not the case. But my comments are still valid. All you need to do is name your routes accordingly.